registration/registration-services/src/main/java/io/mosip/registration/dao/impl/RegistrationDAOImpl.java

```diff
@@ -159,14 +159,14 @@ private String getKey(String groupName) throws RegBaseCheckedException {
 
 	private String getAdditionalInfo(Object fieldValue) {
 		String value = null;
-		if (fieldValue instanceof List<?>) {
+		if (fieldValue != null && fieldValue instanceof List<?>) {
 			Optional<SimpleDto> demoValueInRequiredLang = ((List<SimpleDto>) fieldValue).stream()
 					.filter(valueDTO -> valueDTO.getLanguage().equals(ApplicationContext.applicationLanguage())).findFirst();
 
 			if (demoValueInRequiredLang.isPresent() && demoValueInRequiredLang.get().getValue() != null) {
 				value = demoValueInRequiredLang.get().getValue();
 			}
-		} else if (fieldValue instanceof String) {
+		} else if (fieldValue != null && fieldValue instanceof String) {

```

Comment:
can we not return on the top if null so the if conditions look neat. also can you avoid else?